jit/src/lib.rs

```diff
@@ -165,6 +180,7 @@ pub enum JitType {
     Int,
     Float,
     Bool,
+    PrintFunction,
```

Comment:
Would it make sense to represent this as `Function(enum JITFunction)` or something along those lines?

Reply:
I think that would make sense when adding a second builtin function. See also #4263.

Reply:
Yeah, sounds good 👍